config/grad.uiowa.edu/views.view.srop_scholars.yml

```diff
@@ -1,6 +1,6 @@
 uuid: 7138678e-9e96-4ebc-b20b-578be83126d5
 langcode: en
-status: true
+status: false
```

Comment:
Did you mean to set this back to false after https://github.com/uiowa/uiowa/pull/2923?

Reply:
Nope

Reply:
ok, I'll fix it.